lib/orocos/process.rb

```diff
@@ -990,14 +1007,6 @@ def spawn(log_level: nil, working_directory: Orocos.default_working_directory,
             name_mappings = prefix_mappings.merge(self.name_mappings)
             self.name_mappings = name_mappings
 
-            # If possible, check that we won't clash with an already running
-            # process
-            task_names.each do |name|
-                if name_service.task_reachable?(name)
-                    raise ArgumentError, "there is already a running task called #{name}, are you starting the same component twice ?"
-                end
-            end
-
```

Comment:
is there no risk anymore to start the same component twice?

Reply:
The point is to not use the name services anymore